dashboard/config/linux/bits/subsystems.yml

```diff
@@ -532,9 +532,9 @@ config:
  - BIG_KEYS: [-kmsan]
  - TRUSTED_KEYS
  - KEY_DH_OPERATIONS
- - ASYMMETRIC_TPM_KEY_SUBTYPE
+ - ASYMMETRIC_TPM_KEY_SUBTYPE: [-v5.18]
  - PKCS8_PRIVATE_KEY_PARSER
- - TPM_KEY_PARSER
+ - TPM_KEY_PARSER: [-v5.18]
```

Comment:
Where all of these `-v5.18` removed? Or what happened to them?

Reply:
All of them generates "OPTION_NAME is not present in the final config".

Reply:
Added comments to the code.